source/fundamentals/aggregation/aggregation-expression-operations.txt

```diff
@@ -28,5 +27,78 @@ more naturally readable.
 Operations
 ----------
 
+The following sections provide information and examples for
+aggregation expression operations available in the driver.
+The operations are categorized by purpose and functionality.
+
+.. important::
+
+   The driver generates a Query API expression that may be different
+   from the Query API expression provided in each example. However,
+   the expressions are equivalent when evaluated.
```

Comment:
```suggestion
.. note::

   The driver generates a Query API expression that may be different
   from the Query API expression provided in each example. However,
   the expressions will produce the same results.
```

Unsure if this is important to know this before using, rather than a "note".

You may mention that this is "because the driver may perform post-processing or other optimizations" to reassure users, but maybe this is too much detail.

Reply:
Good point. converting to note and adding an appropriate amount of explanation.